code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -661,6 +661,14 @@
 	. = 1
 	..()
 
+/datum/reagent/toxin/capilletum
+	name = "Capilletum"
+	description = "A powerful toxin that exemplifies the patterns of punctured skin, matching their pigments and shapes, and then expands them across the body. Unlike other toxins, it does not have any negative effects."
```

Comment:
So why is this a toxin if it has no negative side effects? 
Toxins are toxins because they're toxic in some way.

This fits more in line with a drug

Reply:
it was originally a toxin so i just put it in there happy to move it